components/array-object-table/src/test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Component from '.';
+
+describe('ArrayObjectTable', () => {
+  let fields = [
```

Comment:
i'm not keen on this style of declaring variables at the top of the describe, and then using them inside individual tests below.

for instance, `fields` here ends up getting redefined later - i can guess that's going to be the case as it's defined with a `let`, but have to trawl down below to see where that happens.  redefining `fields` in a test below also means that other tests after it's redefined need to be aware that it's been redefined.

if there are constant definitions of things to be used through multiple tests then i'd prefer to see them raised outside the `describe` block and declared as `const`, preferably given a slightly more descriptive variable name if there's going to be different versions used

similarly there's not much advantage on declaring a single `wrapper` here - better for that to be `const wrapper =` inside all the individual tests below.  indeed allowing wrapper to persist through multiple tests means that the order of the tests below becomes important - which makes them harder to reason about

additionally if we wish to test different areas of functionality of a component, we can use multiple `describe` blocks inside the test file